cfme/tests/openstack/cloud/test_instances.py

```diff
@@ -170,6 +170,7 @@ def test_delete_instance(new_instance):
 
 
 def test_list_vms_infra_node(appliance, provider, soft_assert):
+    pytest.skip("Skipping test: hypervisors IPs don't match Hosts IPs")
```

Comment:
**REQUIRED** please use either a `skip` or a `uncollect` mark to disable the test since that avoids fixture setup cost

**OPTIONAL** in order to support future development please add a reference to what needs to be done to sort the situation out